packages/cache-core/src/cacheHandler.ts

```diff
@@ -2,6 +2,7 @@ import path from 'path'
 import cookieParser from 'cookie'
 import parser from 'ua-parser-js'
 import { pathToRegexp, Path } from 'path-to-regexp'
+import md5 from 'md5'
```

Comment:
nodejs has built in hash algorithms https://nodejs.org/docs/v20.15.1/api/crypto.html#cryptocreatehashalgorithm-options

Reply:
Fixed. Thanks